tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -115,7 +116,7 @@ def _prepare_input(self):
         raise RuntimeError("Please provide inputs generation function")
 
 
-def get_params(ie_device=None, precision=None):
+def get_params(ie_device=['CPU'], precision=None):
```

Comment:
please remove this change. We would want to test GPU if we at some point get into CI.

Reply:
sorry, accidentally commited